junit/MatrixTest.java

```diff
@@ -437,6 +437,29 @@ public void testGetException()
         testIndex[1] = -1; 
         matrix.get(testIndex[0], testIndex[1]);
     }
+    /**
+     * testIdentity -Tests the creation of the identity matrix.
+     */
+    @Test
+    public void testIdentity() 
+    {
+        Random rand = new Random();
+        int m = rand.nextInt(100);
+        int n = rand.nextInt(100); 
+        double[][] correctMatrix = new double[m][n];
+        for (int i = 0; i < correctMatrix.length; i++) 
```

Comment:
I've written this same thought somewhere on the open PR #89 , but basically I think we should test against array literals. It does no good to test code against a copy of itself. You only confirm that it is, indeed, a copy.